src/containers/SuperAdminOrganizations.tsx

```diff
@@ -0,0 +1,180 @@
+import { Theme, withStyles } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Paper from "@material-ui/core/Paper";
+import DeleteIcon from "@material-ui/icons/Delete";
+import OpenInNewIcon from "@material-ui/icons/OpenInNew";
+import RestoreFromTrashIcon from "@material-ui/icons/RestoreFromTrash";
+import {
+  DataGridPro,
+  GridColDef,
+  GridRenderCellParams
+} from "@mui/x-data-grid-pro";
+import {
+  useEditOrganizationActiveMutation,
+  useGetOrganizationsQuery
+} from "@spoke/spoke-codegen";
+import React, { useState } from "react";
+
+const RedButton = withStyles((theme: Theme) => ({
+  root: {
+    color: theme.palette.warning.contrastText,
+    backgroundColor: theme.palette.warning.main,
+    "&:hover": {
+      backgroundColor: theme.palette.warning.dark
+    }
+  }
+}))(Button);
+
+const SuperAdminOrganizations: React.FC = (_props) => {
+  const {
+    data: organizationsData,
+    loading,
+    refetch: refetchOrganizations
+  } = useGetOrganizationsQuery();
+  const [organizationIdToEdit, setOrganizationIdToEdit] = useState<
+    string | null | undefined
+  >();
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
+  const [organizationActiveType, setOrganizationActiveType] = useState<
```

Comment:
Suggestion for consistency with `organizationIdToEdit`
```suggestion
  const [organizationActiveEdit, setOrganizationActiveEdit] = useState<
```